dom-weather-app/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node
+ENV NODE_ENV=production
+WORKDIR /dom-weather-app
+COPY ["package.json", "package-lock.json*", "./"]
+RUN npm install --production
+COPY . .
+CMD ["node", "server.js"]
```

Comment:
Ten cały server teraz jets potrzebny tylko do robienia ajaxa (bo nie da się jak html otworzysz z file://xxx), są też gotowce jak [http-serve](https://www.npmjs.com/package/http-server), ale na potrzeby ćwiczenia spoko

Reply:
No kumam - dzięki temu chociaż troche liznąłem na czym to polega